google-cloud-spanner/src/test/java/com/google/cloud/spanner/AsyncTransactionManagerTest.java

```diff
@@ -1112,4 +1113,56 @@ public void asyncTransactionManagerQuery() throws Exception {
       }
     }
   }
+
+  @Test
+  public void asyncTransactionManager_shouldPropagateStatementFailure()
+      throws ExecutionException, InterruptedException, TimeoutException {
+    final Statement garbledStatement =
+        Statement.newBuilder("INSERT INTO BOOKS (UUID, TITLE) VALUES ('123', 'Test book')jljlk")
```

Comment:
Optional: use a table more commonly found in client library integration test, so future readers won't have to wonder where `BOOKS` came from.